harvard-university-of-leeds.csl

```diff
@@ -32,7 +36,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="text" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
+      <name and="text" delimiter-precedes-last="never" et-al-min="0" initialize-with="." name-as-sort-order="all"/>
```

Comment:
> But it appears to be more robust to just set it to a high number

That's just because it prevents all authors to be printed for silly papers with thousands of authors, right? (see http://www.nature.com/news/physics-paper-sets-record-with-more-than-5-000-authors-1.17567 )